package.json

```diff
@@ -149,5 +149,8 @@
     "type-fest": "4.25.0",
     "typescript": "5.4.5",
     "webpack-bundle-analyzer": "4.10.2"
+  },
+  "volta": {
+    "node": "20.18.1"
```

Comment:
Wrong commit, let's stick this and the CI change together. I'll discuss Volta in general in overall review comment